### ISSUE TITLE
Do you take results for older versions of android?

### ISSUE THREAD

**Issue by antoniusf** [accessibilitysupported/a11ysupport.io#142]
Hi, I have a phone with android 8 (but otherwise up-to-date software), and I can't replicate the results you got for android 10. ("aria-pressed" is not announced correctly when focusing the buttons.) I can send the test report if you want, but I wasn't sure if you were even interested in that? I'm sorry if this is unnecessary.

(I don't regularly use a screen reader so maybe I did something wrong, too, I don't know.)

**Comment by mfairchild365** [comment]
Hi @antoniusf - thank you for opening this issue. Is this in reference to this test? https://a11ysupport.io/tests/tech__aria__aria-pressed

Can you tell me what you are observing (actual output from the screen reader) and what you expected?

**Comment by antoniusf** [comment]
Hi @mfairchild365, thanks so much for answering! Yes, that's the test I was referring to, sorry for not being clearer.

What I'm observing: All three buttons (false, true, mixed) are announced as "action, toggle button", regardless of their state, when I navigate to them.

What I expected: Something like the examples on your test page, that I also get an indication of the current state of the button.

**Comment by antoniusf** [comment]
I don't hear any form of "not checked" or "checked".

(To be clear, the complete output from the screen reader sounds more like "action, toggle button, double tap to activate, double tap and hold for long press". I'd omitted the last part since it wasn't in the other testing protocol either.)

**Comment by mfairchild365** [comment]
And just to confirm, what browser are you using?

**Comment by antoniusf** [comment]
This was Chrome 86 on Android.

**Comment by mfairchild365** [comment]
Okay, I can replicate your results now. That was strange, I was still hearing 'checked' and 'not checked' until I restarted by device. Now I don't hear any state.

Chrome 86
Android accessibility suite 8.2
Android 7.0

I'd like to verify that this is actually working in Android 10. My guess is that an update to Chrome or Talkback broke this, and it isn't working in Android 10 now.

**Comment by antoniusf** [comment]
Thanks for your effort! I'm glad you could replicate the problem. Strange that it stopped working for you, maybe you're right that an update broke something?

Unfortunately, I don't own any devices with Android 10, so I can't help verify that :/ I'm also very curious if it's now broken there, too.

**Comment by mfairchild365** [comment]
I confirmed that the issue exists in Android 10 and 11. I'll update the results.

**Comment by antoniusf** [comment]
Thanks for confirming this! I hope that means that this is a bug in talkback and google will fix it at some point.

I think I'm going to close this issue now, thank you so much for your help!

**Comment by mfairchild365** [comment]
fixed results in https://github.com/accessibilitysupported/a11ysupport.io/commit/919dfe456e391e83ff5a91e38810559da8e66ae1
